script/lock_user

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+if ARGV.empty?
+  puts "Locks a user account so it can't be used."
+  puts "USAGE: script/malicious_user USERNAME"
+  exit
+end
+
+handle = ARGV.first
+puts "Locking user #{handle}..."
+
+ENV["RAILS_ENV"] ||= "production"
+require_relative "../config/environment"
+
+user = User.where(handle: handle).first
+
+user.update_attribute(:email, "security+locked-#{handle}@rubygems.org")
```

Comment:
While this blocks the handle and they lose access to already published gem, I was hoping we could block their email as well. We could add a new column to mark blocked users and use the column in [a sign in guard](https://github.com/rubygems/rubygems.org/blob/master/lib/confirmed_user_guard.rb#L1).

Reply:
Every account I have seen with malicious gems has an e-mail from a throwaway service like mailinator or sharklasers. I think it would be a lot of work to add individual email blocking, and probably not help us since I think no one uses the same email address twice.